modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -11,7 +11,7 @@
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
-
+namespace LORIS\conflict_resolver;
```

Comment:
I think there's something wrong with this PR. This (and a bunch of files below) are unrelated and seem to be from something else..

Reply:
@driusan 
I messed up in this branch.
I will close this and send a new one.